tests/components/Translation.js

```diff
@@ -0,0 +1,61 @@
+describe("Translation", function() {
```

Comment:
I believe Cypress test files should have `*-cy.js` postfix otherwise they won't run